### DISCUSSION TITLE
kick

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#94] [Ideas]
Добавить метод kick() в Api игры. Метод будет исключать игрока из сервера с выводом ему текста. Добавить свойство PossibilityConnect у игрока в Api, свойство будет разрешать игроку заходить на сервер.

**Comment by kkohno** [comment]
киками можно испортить другому игроку игру... тут просто так в апи кик вставлять не можем. все условия кика, если такой и будет, будут прописаны до начала катки в игре, чтобы небыло возможности сделать кик игроку со спецправами

**Comment by KukushkaSex** [comment]
Пусть они сами уже ограничивают этот Кик

**Comment by forwork10** [comment]
ты лох, не мечтай даже

**Comment by kkohno** [comment]
по кикам - там идея есть такая. что просто инициализируется голосование на кик и далее все уже решают выкинуть или нет

**Comment by KukushkaSex** [comment]
Такое себе, лучше все таки в конфиге режима описать условия...
